### PR TITLE
simif_f1_t::fpga_setup prints AGFI ID at simulation init time

### PR DESCRIPTION
**Pull request by timsnyder-siv**
This will make it so that the uartlog will contain the loaded AGFI ID
at the beginning.  Before this addition, one needed to trace the breadcrumbs through
firesim manager logfiles to get to the infrasetup output.  Should make it
easier to audit and reproduce results.

### CODE REVIEW

**Comment by timsnyder-siv** [comment]
@davidbiancolin , you and I talked about this at one point but I don't see an issue about it.  I finally got around to looking at the source code for `fpga-describe-local-image` in the AWS FPGA SDK to see how it uses the fpga_mgmt library to look it up and added a minimal change that didn't need to actually popen the `fpga-describe-local-image` to look it up.  See https://github.com/aws/aws-fpga/blob/f063204ffcb306a2ea85f68e775ccbd759969628/sdk/userspace/fpga_mgmt_tools/src/fpga_local_cmd.c#L120-L121 for reference.

**Comment by timsnyder-siv** [comment]
oops, looks like basing on 1.11 the CI doesn't build anymore.  I'll cherry-pick the fix and repush.

**Review by davidbiancolin — APPROVED** [review]
lgtm

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   main-env:
     docker:
-      - image: firesim/firesim-ci:v1.0
+      - image: firesim/firesim-ci:v1.1
```
[the remark this answers is not in the document]
Comment (davidbiancolin) [reply .circleci/config.yml:6]: nvm...

**Pull request merged**